src/config.ts

```diff
@@ -20,6 +20,7 @@ export default {
     noBinary: true,
     allow: [
       'add',
+      'ball',
```

Comment:
I do not think this is the right config for AlexJS, please check their documentation

Reply:
I think this is correct, coz words like just, simple and master also exist here, if groups like dad-mon and host-hostess do.